scripts/mkcloud

```diff
@@ -428,10 +431,15 @@ function onhost_deploy_image()
     local role=$1
     local image=$(dist_to_image_name $2)
     local disk=$3
+    local image_url="http://$clouddata/images/$arch/$image"
+
+    [[ -n $want_efi ]] && {
+        image_url="http://download.suse.de/ibs/Devel:/Cloud:/Images/images_SLE_12_SP2/mkcloud-admin-node-SLE12SP2-uefi.x86_64.qcow2"
+        image="mkcloud-admin-node-SLE12SP2-uefi.x86_64.qcow2"
```

Comment:
@bmwiedemann: The SP1 image cannot be built currently due to some unsupported dependencies (libsystemd, rubygem-cfa). So currently only SP2 image is possible.